Tests/SWBTaskExecutionTests/PBXCpTests.swift

```diff
@@ -424,8 +424,10 @@ fileprivate struct PBXCpTests: CoreBasedTests {
                 #expect(result.output == (
                     "copying \(fwkName)/...\n"
                 ))
+   #if !os(Windows)
```

Comment:
Please don't use build-time conditions except for platform-specific APIs; this helps maximize compile testing of every code path across different platforms which is useful when refactoring, etc. Instead you can use:

```swift
try ProcessInfo.processInfo.hostOperatingSystem() != .windows
```